src/tests/requirements-test.txt

```diff
@@ -1,4 +1,4 @@
-apache-airflow==2.5.3
+apache-airflow==2.8.0
```

Comment:
@latika-wadhwa is there a reason we use 2.8.0, considering Composer doesn't support this version?

Reply:
@dmedora I had [1] issues with version 2.5.3/2.6.3 that seems to be supported in 2.8.0 version. Missed checking for composer. Let me revert it back.

[1] -
 TIMEZONE = pendulum.tz.timezone("UTC")
E   TypeError: 'module' object is not callable